### ISSUE TITLE
Custom window button theme not applying on plasma-git installation

### ISSUE THREAD

**Issue by dr460nf1r3** [psifidotos/applet-window-buttons#155]
Hey, the applet recently started to fail showing custom window button themes with the `current` option chosen. Instead, it shows the Breeze buttons. 

![image](https://user-images.githubusercontent.com/12834713/144868400-d5b39cf3-9d60-48e4-8e7a-a81e13605f5d.png)

**Comment by psifidotos** [comment]
What plasma version? Upcoming 5.24?

**Comment by dr460nf1r3** [comment]
Its a full KDE/Plasma-git (master branch) installation on Garuda Linux.

**Comment by psifidotos** [comment]
Garuda is using Plasma Git version?

**Comment by dr460nf1r3** [comment]
The official images aren't, but we are currently working to get a vanilla kde-git image out soon - I'm using the master branch packages on my device myself as well :)

**Comment by psifidotos** [comment]
If you set the correct theme from the list, does it work properly?

**Comment by psifidotos** [comment]
no, looks like a big breakage... that probably needs 0.11 version that will require plasma 5.24

**Comment by dr460nf1r3** [comment]
Alright then. Thanks for your work! :3

**Comment by psifidotos** [comment]
I tried in my plasma git version system, I didnt manage to reproduce... it might be something very new...

**Comment by TheBigWazz** [comment]
Updated to Plasma 5.24 today and am getting the same result, with the decoration defaulting to Breeze when 'Current' is selected. Custom decorations were working before the update on 5.23 Only 'Current' and 'Breeze' show up in the Decorations drop-down menu, when I have several Aurorae decorations installed.

**Comment by steveroch-rs** [comment]
I have the same problem. I upgraded to 5.24 today and the applet defaults to Breeze theme. I have "Current" selected and there's also "Oxygen" as an option but all 3 default to the same Breeze icons.

**Comment by sarvasana** [comment]
Before everybody starts replying they are having the same issue. Please just provide a thumbs up on one of the previous people mentioning they have this issue also on 5.24.

**Comment by michaldybczak** [comment]
I can confirm that the bug is fixed. I just noticed a new Plasma version 5.24.2 today, so I update it, then updated latte, then I updated all latte plasma5-applets git, and then restarted latte. All went back to normal and after switching to aurorae theme, I see correct buttons from it.

The only difference is, before I left the size from the theme, and it looked exactly as on the titlebar, now it looked different, too huge, no spacing, etc., so I had to play with the settings to find a proper look, but eventually I did, so all is OK.

Thank you @psifidotos!

**Comment by gamerua** [comment]
@psifidotos I have this problem, but not with latte panel. With standard kde panel in decoration field only breeze. 
kde: 5.25.5. Debian Testing. Latest version of applet.
